service/controller/resource/app/resource.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/cluster-operator/service/internal/basedomain"
```

Comment:
It looks like all changes in this file can be reverted since we just dropped the code in `current.go` completely.